### DISCUSSION TITLE
Server detected but the plugin doesn't connect to it

### DISCUSSION THREAD

**Discussion by MrPisquan** [apohl79/audiogridder#802] [General]
Hi,

I recently changed my setup adopting all wired connections.

After that, AG plugin doesn't connect anymore to the server, even if the server is clearly detected in the list of servers available (3 servers available for the same machine - very weird indeed).

Please find attached the log files.

Thank you for your help!!!

[AudioGridderServer_2021-11-18_14-22-28.log](https://github.com/apohl79/audiogridder/files/7570340/AudioGridderServer_2021-11-18_14-22-28.log)
[AudioGridderServer_2021-11-19_10-52-50.log](https://github.com/apohl79/audiogridder/files/7570341/AudioGridderServer_2021-11-19_10-52-50.log)
[AudioGridderServer_2021-11-19_10-54-41.log](https://github.com/apohl79/audiogridder/files/7570342/AudioGridderServer_2021-11-19_10-54-41.log)
[AudioGridderPlugin_2021-11-18_14-23-31.log](https://github.com/apohl79/audiogridder/files/7570343/AudioGridderPlugin_2021-11-18_14-23-31.log)
[AudioGridderPlugin_2021-11-19_12-39-11.log](https://github.com/apohl79/audiogridder/files/7570344/AudioGridderPlugin_2021-11-19_12-39-11.log)
![IMG_7759](https://user-images.githubusercontent.com/75130976/142626606-ffd3c221-01de-41d2-b5fe-6f56b3bee3a0.jpg)

**Comment by MrPisquan** [comment]
Hello,
unfortunately I can't find a way out from this problem. Has any one experienced a similar situation?
Andreas, may you give me some suggestions please?
Thank you for your help and your time.

**Reply by apohl79** [reply]
For some reason, you only see the ipv6 addresses. I assume you also assign ipv4 addresses to your computers in your network? If yes, can you try to connect by manually setting the IP?

**Reply by MrPisquan** [reply]
Hi Andreas,
thank you for reaching out!
Yes, the ipv4 are assigned. I tried to connect the plugin using the ipv4 address but with no luck. I am using AG a lot - it is so good that it  became central in my productions - so I'm stuck! 
Thank you for your help.

**Reply by apohl79** [reply]
Have you tried to disable your firewall?

**Reply by MrPisquan** [reply]
yes it's disabled. Also, there is some weirdness in the behaviour of the server. It doesn't retain the settings. I am trying to change the preferences (enabling/disabling sandbox, parallel loading etc), but at any restart the settings are the same.

**Reply by MrPisquan** [reply]
Sorry, now the settings are retained - I had to wait the scanning of the plugins

**Reply by MrPisquan** [reply]
Thanks to your input, I disabled the ipv6 protocol in the network settings, and now it works! I am so relieved! AG is a blessing. Really. 
Can't wait the future of it, it must be adopted in every studio!
Thank you again for the support and the help.
